lib/driver.js

```diff
@@ -115,12 +115,14 @@ class YouiEngineDriver extends BaseDriver {
             }
             await this.device.startSession(caps, this);
             break;
+          }
           case "noproxy":
           case "connecttoapp":
             break;
-          default:
+          default: {
             let msg = `Unsupported platformName: ${caps.platformName}`;
             logger.errorAndThrow(msg);
```

Comment:
same with single line
default: 
logger.errorAndThrow(`Unsupported platformName: ${caps.platformName}`); 

(no braces needed because there's no var declaration.)